sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -314,7 +314,7 @@ Arguments: HashedRelationBroadcastMode(List(cast(input[0, int, true] as bigint))
 Left keys [1]: [key#x]
 Right keys [1]: [key#x]
 Join condition: None
-       
```

Comment:
does it mean the master branch outputs some extra spaces in EXPLAIN FORMATTED?

Reply:
It turns out that we misused `stripMargin` for explained nodes. For example, in the case of `Join`,

https://github.com/apache/spark/blob/f543d6a1ee76de8cae417ff480fa9c0e0ce5343d/sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BaseJoinExec.scala#L50-L53

We actually should use `|""".stripMargin` to exclude the extra spaces.

I'll send a PR later to fix the issue.